shipyard/manifest.json

```
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3ea4d492-9d1c-4dda-bb8a-e571eca3ce7d",
+  "app_id": "shipyard",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Shipyard logs to Datadog dashboard",
+    "title": "Shipyard",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Datadog connection settings in your Shipyard dashboard",
+        "image_url": "images/datadog-input.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Shipyard logo in teal",
+        "image_url": "images/shipyard-teal.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Containers",
+      "Category::Developer Tools",
+      "Category::Testing",
+      "Submitted Data Type::Logs",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "Shipyard Sample Dashboard": "assets/dashboards/shipyard_overview.json"},
+    "monitors": {},
+    "logs": {},
+    "saved_views": {}
+  },
+  "author": {
+    "support_email": "support@shipyard.build",
+    "name": "Shipyard Support",
+    "homepage": "https://shipyard.build/",
+    "sales_email": "sales@shipyard.build"
+  },
+  "oauth": {},
```

```suggestion
  "oauth": {}
```
Turns out this was the actual cause of the failing validations.